apps/frontend/src/app/3_organisms/ZeroLocForm/components/FormContent.tsx

```diff
@@ -41,6 +41,7 @@ import { AmountType } from '../types';
 export type OpenTroveProps = {
   hasTrove: false;
   liquidationReserve: Decimal;
+  debtType: AmountType;
```

Comment:
When opening trove, `hasTrove` is always `false` and always `true` for Adjusting.
So it means that you don't need to add `debtType` here, `hasTrove == false` is same as `debtType == AmountType.Add`